cmd/oteltestbedcol/go.mod

```diff
@@ -2,7 +2,7 @@
 
 module github.com/open-telemetry/opentelemetry-collector-contrib/cmd/oteltestbedcol
 
-go 1.19
+go 1.20
```

Comment:
lint is failing on this, running `make genotelcontribcol` reverts this change to go 1.19
How to fix linting here?

Reply:
I'd suggest looking at the same PR done for the 1.19 and see what's missed

Reply:
Problem solved, had to bump `go.opentelemetry.io/collector/cmd/builder` to latest sha of master.